internal/tls/common.go

```diff
@@ -6,6 +6,7 @@ var tlsVersionMap = map[string]uint16{
 	"TLS10": tls.VersionTLS10,
 	"TLS11": tls.VersionTLS11,
 	"TLS12": tls.VersionTLS12,
+	"TLS13": tls.VersionTLS13,
```

Comment:
Are these here so people can manually use TLS13 or the new ciphers?  Are we not defaulting to the newest yet because they're not available in all go versions we target?

Reply:
fyi looks like go 1.12 had support for tls 1.3

Reply:
These are moved over from `common_go112.go`, it is possible to set some advanced TLS options in additiona to the options that are shown in the sample config: https://github.com/influxdata/telegraf/blob/master/docs/TLS.md.  Actually looks like I need to update these docs.